scripts/build_test.py

```diff
@@ -1070,10 +1070,13 @@ def mock_clean() -> None:
         compare_file_count_swap = self.swap(
             build, '_compare_file_count', mock_compare_file_count)
         clean_swap = self.swap(build, 'clean', mock_clean)
+        install_python_dev_dependencies_swap = self.swap_to_always_return(
+            install_python_dev_dependencies, 'main', None
+        )
```

Comment:
Can we assert that `install_python_dev_dependencies.main()` gets called correctly?

Reply:
Done.